CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
```

Comment:
This is high level contributing guidance. Each sub project should includes a `DEVELOPMENT.md` for detail development and testing guidance.